services/dy-static-file-server/docker/custom/Dockerfile

```diff
@@ -3,18 +3,35 @@ FROM nginx:1.21.0-alpine as base
 #
 #  USAGE:
 #     cd services/dy-static-file-server
-#     docker build -f Dockerfile -t dy-static-file-server:prod --target production ../../
+#     docker build -f docker/custom/Dockerfile -t dy-static-file-server:prod  .
 #     docker run dy-static-file-server:prod
 #
 
 LABEL maintainer=sanderegg
 COPY docker/custom/nginx.conf /etc/nginx/nginx.conf
 COPY docker/custom/boot.sh /boot.sh
 
-COPY src/dy_static_file_server/hello-world.txt /www/hello-world.txt
+RUN apk add --update --no-cache "python3=3.8.10-r0" py3-pip
+RUN pip3 install --upgrade pip virtualenv
+
+RUN mkdir -p /venv
+RUN python3 -m venv /venv
+
+# add python app requirements
+COPY requirements_app.txt /tmp/requirements_app.txt
+RUN /venv/bin/pip3 install -r /tmp/requirements_app.txt
+
+COPY static-content/hello-world.txt /www/hello-world.txt
+RUN mkdir -p /www/inputs
+RUN mkdir -p /www/outputs
+
+RUN mkdir -p /workdir
```

Comment:
I would recommend using ARG there as well. it's more flexible later
and have 1 RUN call (minimize number of layers)
```
RUN mkdir -p ${INPUT_DIR} \
         && mkdir -p ${OUTPUT_DIR} \
         && mkdir -p ${WORK_DIR}
```